dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -282,8 +295,12 @@ def define_graph(self):
 
 READ_SHUFFLE=args.read_shuffle
 
-print("GPUs: {}, batch: {}, workers: {}, prefetch depth: {}, loging interval: {}, fp16: {}, NHWC: {}, READ_SHUFFLE: {}"
-      .format(N, BATCH_SIZE, WORKERS, PREFETCH, LOG_INTERVAL, FP16, NHWC, READ_SHUFFLE))
+SMALL_DATA_SET = args.small
+if SMALL_DATA_SET:
+    test_data = small_test_data
+
+print("GPUs: {}, batch: {}, workers: {}, prefetch depth: {}, loging interval: {}, fp16: {}, NHWC: {}, READ_SHUFFLE: {}, small dataset: {},"
+      .format(N, BATCH_SIZE, WORKERS, PREFETCH, LOG_INTERVAL, FP16, NHWC, SMALL_DATA_SET, READ_SHUFFLE))
```

Comment:
The format string and printed arguments are reversed for the last two.

Reply:
Done